dhall-toml/src/Dhall/Toml.hs

```diff
@@ -7,33 +7,96 @@ module Dhall.Toml
     , dhallToToml
     ) where
 
-
+import Control.Monad     (foldM)
 import Control.Exception (Exception)
+import Data.Text         (unpack)
 import Data.Void         (Void)
-import Dhall.Core        (Expr)
+import Dhall.Core        (Expr, DhallDouble(..))
 import Dhall.Parser      (Src)
 import Toml.Type.TOML    (TOML)
+import Toml.Type.Key     (Piece(Piece), Key(Key))
+import Toml.Type.Printer (pretty)
+
+import qualified Data.List.NonEmpty as NonEmpty
+import qualified Data.Text.IO       as Text.IO
+import qualified Dhall.Map          as Map
+import qualified Dhall.Core         as Core
+import qualified Dhall.Import
+import qualified Dhall.Parser
+import qualified Dhall.TypeCheck
+import qualified Toml.Type.TOML     as T
+import qualified Toml.Type.Value    as TVal
```

Comment:
I'd appreciate somewhat more expressive abbreviations, for example
```suggestion
import qualified Toml.Type.TOML     as Toml.TOML
import qualified Toml.Type.Value    as Toml.Value
```

Or maybe just use the fully qualified names.